app/templates/partials/applications_table.html

```diff
@@ -47,6 +47,9 @@
         {
             'html': application.reference
         },
+        {
+            'html': application.language
```

Comment:
Not really cause of your PR, but do you know why these are `html` as opposed to `text` - guess it doesn't change much just curious, feel free to ignore this.